dipy/stats/__init__.py

```diff
@@ -4,4 +4,4 @@
 w_string = "The `dipy.stats` module is still under heavy development "
 w_string += "and functionality, as well as the API is likely to change "
 w_string += "in future versions of the software"
-warnings.warn(w_string)
+# warnings.warn(w_string)
```

Comment:
I think you should either remove the whole block or keep it. @arokem did you totally finish with the design of afq_profile too? If yes, just remove these lines

Reply:
removed